docs/organizations/using-a-coding-standard.md

```diff
@@ -9,6 +9,8 @@ Create a coding standard on your organization to define and apply shared tool an
 
 The organization coding standard helps you ensure that Codacy analyzes multiple repositories with the same tool and code pattern settings. For example, you can use the organization coding standard to ensure that all your repositories and teams are following the same security rules or coding conventions.
 
+If a coding standard doesn't include a tool, any pre-existing repository settings for that tool aren't modified by the coding standard.
```

Comment:
The sentence looks very clear to me. However, it seems like a small detail lost inside this concept piece.
Would it make sense to move this sentence inside the "Important box" below? It would fit as the first bullet of a two-bullet list?
WDYT?

Reply:
Good point. I agree 👍